index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Bus Mall Focus Group</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <section>
+      <div id="picture-box">
```

Comment:
Why does this div exist if it is the only child of section?  You can just move the id to section, fix the 2 lines of CSS that will need to be moved / updated and have the page displayed the same.